packages/atlas-experiment-table/src/TableCell.js

```diff
@@ -11,6 +11,10 @@ const TableCell = ({ dataRow, dataKey, image, linkTo, host, width }) => {
     if (image[dataRow[dataKey]]) {
       cellItem = <img {...image[dataRow[dataKey]]}/>
     }
+    else if (Object.keys(image).includes(`anndata`)) {
+      let ifAnndata = dataRow[dataKey].substring(0,6) === `E-ANND` ? `anndata` : `nonAnndata`
```

Comment:
I believe this check can be improved using a regular expression as compared to the hard-coding substring index. I guess you just want to see if the experiment accession has `E-ANND` in it? if yes, have a look [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) to see if this check can be implemented using regex.